Topics/Tech_Stacks/NumPy.md

```diff
@@ -0,0 +1,199 @@
+# Learning NumPy
```

Comment:
It may or may not be helpful to indicate in the title of the file that this is related to Python although the name includes Py so it does suggest Python. If there was a Python document I would've recommended you just add it there.